packages/wonder-stuff-server-google/src/add-app-engine-middleware.js

```diff
@@ -0,0 +1,34 @@
+// @flow
```

Comment:
Copied from render-gateway and updated to reflect the pieces that are now in the common Wonder Stuff Server instead